New Drink: The Himalayan

Sneeker134 opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

Adds a drink to the bar, the Himalayan, which rapidly freezes the drinker and has the potential to slowly rack up damage through "brain freezes." Made from ice, frost oil, vodka, and tonic water.

![himalayan](https://user-images.githubusercontent.com/53362182/73139757-e1449000-4036-11ea-9fe3-e31f5e0827fb.gif)

## Why It's Good For The Game

I was going through reagents generated through botany that the bar doesn't have a good drink for yet, and stumbled across Frost Oil. You can make iced beer with it but that drink didn't seem to do anything particularly unique. So, I decided to add this drink! It rapidly cools you down, and has a small chance to give you a brain freeze, which inflicts some minor brain damage. Also does the ice watcher thing of freezing you solid. Adding more stuff for bartenders to make is always fun, and adding more reason for bartender to interact with other departments helps them do more besides sit behind their counter the whole shift.

My only real concern is the rate of the brain damage. I don't think this drink should really be viable as a poison, but if you drink a ridiculous amount of it I could see being able to justify it having some more lasting effects. I'm not really sure how much brain damage this realistically inflicts, so any feedback on that part would be welcome. The art also isn't great, but I'm not a pixel artist so...

## Changelog
:cl:
add: New drink, the Himalayan. Rapidly cools down whoever consumes it,  potentially giving them a brain freeze!
/:cl:

Sneeker134 committed:
Merge pull request #4 from tgstation/master

merge

Sneeker134 committed:
Finished Himalayan.

Arathianen commented:
Name is lizardbane instead and we have a deal.

evilcyborgmrt commented:
More drinks is ok. More drinks that actually do things is good.

Sneeker134 committed:
Forgot trailing new line.

FantasticFwoosh commented:
Reading this i thought it would make you intake less O2 to breathe in thin air, but whatever this is neat too.

Sneeker134 committed:
Changed recipe to actually work.

Didn't know the Vodka tonic was a drink, so changed the base recipe to include that instead of each unit individually.

Sneeker134 committed:
A Vodka Tonic is only one word I guess.

fml

wesoda25 commented:
Impedrezene is a pretty weak brain toxin, and it has a 80% chance each tick of giving 2 brain damage. So this probably won't be _that_ bad.

Might wanna dial it back though so that people are more willing to drink it, and it becomes less of a grief drink.

Sneeker134 commented:
> Impedrezene is a pretty weak brain toxin, and it has a 80% chance each tick of giving 2 brain damage. So this probably won't be _that_ bad.
> 
> Might wanna dial it back though so that people are more willing to drink it, and it becomes less of a grief drink.

1/16 the power of a pretty weak brain toxin seems like it probably won't end up doing much, but it seems high enough that if you drank a gigantic amount then you might start running into some issues, which is where I personally think the sweet spot is out. Its not like there is much reason to make this except for the bounty anyway, though I suppose it could be useful in keeping your temperature low if you plan on doing something really stupid.

Iatots commented:
Maybe the brain damage could be dependant on your temperature?

Sneeker134 commented:
> Maybe the brain damage could be dependent on your temperature?

Seems like an interesting idea and has potential meme dream potential with a cryo-sting bartender. I don't really like the idea that people using this to cool themselves down wouldn't really end up suffering much of the effects from it though either. I'll consider it, but I kind of like it affecting everyone evenly, since lizard men are already going to get boned with the temperature drop anyway; double dipping on the brain damage seems kind of cruel.

fludd12 commented:
Make its' overdose be removing all of itself from your blood and freezing you in a solid block of ice like icewatchers can.

Sneeker134 commented:
> Make its' overdose be removing all of itself from your blood and freezing you in a solid block of ice like icewatchers can.

I didn't even know that _was_ a status effect! I think having everything tied together is pretty nice, so instead of an overdose effect I'm just gonna tie it up with the brain freeze. I'll try and add it in and test it out. Thanks for the idea!

Sneeker134 committed:
Added freeze effect.

Now applies the icewatcher ice cube freeze on the proc. Also, I updated the hexcolor to actually be blue.

fludd12 commented on code/modules/reagents/chemistry/reagents/alcohol_reagents.dm:
```diff
@@ -2178,18 +2178,19 @@ All effects don't start immediately, but rather get worse over time; the rate is
 		M.adjustOrganLoss(ORGAN_SLOT_BRAIN, 2)
-		to_chat(M, "<span class='notice'>["Your head pounds; you have a brain freeze!"]</span>")
+		M.apply_status_effect(/datum/status_effect/freon/watcher)
+		to_chat(M, "<span class='notice'>["Your blood is so cold, it freezes the air around you! Even worse, you have a brain freeze!"]/span>")
```
```suggestion
		to_chat(M, "<span class='notice'>"Your blood is so cold, it freezes the air around you! Even worse, you have a brain freeze!"</span>")
```
You don't need the square brackets (I think?) but notably, you were missing the opening triangle bracket to the closing span tag.

Sneeker134 committed:
Update code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

bracket bad triangle good

Co-Authored-By: Fel <fludd121@gmail.com>

Sneeker134 committed:
turns out you do need brackets

XDTM commented on code/modules/reagents/chemistry/reagents/alcohol_reagents.dm:
```diff
@@ -2173,3 +2173,24 @@ All effects don't start immediately, but rather get worse over time; the rate is
+	name = "Himalayan"
+	description = "Freezing cold; its a miracle"
+	boozepwr = 40
+	color = "##0000ff"
```
one # too many, although it could still be functional

Sneeker134 replied:
Must've not deleted the hastag when copy pasting the new color in, and I don't think how it looked in a syringe or beaker is something I tested in game. Oops!

Sneeker134 committed:
RIP extra hashtag

Sneeker134 committed:
yolo attempt to fix merge conflicts

Sneeker134 committed:
yolo merge fix attempt

Sneeker134 commented:
I'm too small brain to merge, consider the PR dead. Sorry. :(
